api/DpiAware.md

```diff
@@ -5,6 +5,8 @@
 
 Available only on Windows. All methods of this class are static, access them through [cefpython](cefpython.md).`WindowUtils`.
 
+Usage of this class is not encouraged, as upstream chromium is already support high-dpi by default
```

Comment:
Remove that sentence. Scale and GetSystemDpi are still needed to set appropriate window sizes.